src/Compilers/CSharp/Portable/Symbols/Source/TypeParameterConstraintClause.cs

```diff
@@ -20,6 +20,13 @@ internal enum TypeParameterConstraintKind
         Unmanaged = 0x08,
         NullableReferenceType = ReferenceType | 0x10,
         NotNullableReferenceType = ReferenceType | 0x20,
+
+        /// <summary>
+        /// Type parameter has no type constraints, including `struct`, `class`, `unmanaged` and is declared in a context 
+        /// where nullable annotations are disabled.
+        /// Cannot be combined with <see cref="ReferenceType"/>, <see cref="ValueType"/> and <see cref="Unmanaged"/>.
+        /// </summary>
+        UnknownNullabilityIfReferenceType = 0x40,
```

Comment:
>UnknownNullabilityIfReferenceType [](start = 8, length = 33)

"Unknown" is now called "Oblivious" in `NullableAnnotation`.  Please rename this to match.